### ISSUE TITLE
Problem when running npm start

### ISSUE THREAD

**Issue by MaybeGoogle** [TheRacingLion/Discord-SelfBot#42]
I have installed everything, npm i worked well. 

I get this issue: 
<img width="565" alt="screen shot 2017-07-14 at 3 55 14 pm" src="https://user-images.githubusercontent.com/30007491/28230529-fecfa826-68ac-11e7-8ed5-8922e2d5bfef.png">

Heres part of the config file, hopefully without identifying information. 
<img width="398" alt="screen shot 2017-07-14 at 3 57 22 pm" src="https://user-images.githubusercontent.com/30007491/28230581-34972ac4-68ad-11e7-9e60-39ee4776a8a3.png">

Thanks

**Comment by MaybeGoogle** [comment]
By looking at some closed issues, I was able to fix this one, by removing the "SERVER ID"s. It now gives me more issues. This one looks like it has to do with default embed color.

**Comment by MaybeGoogle** [comment]
ok, so I copied the raw file and replaced the config file. I fixed some more of my mistakes (eg. different quotation marks, extra comma at end of list, etc.) and it worked. However, I didn't really change the settings the way I wanted in the config file because it was just a test.

Can I change the config file as the bot is running, and have that reflected in the running instance of the bot?

Thanks for reading all of this
